### DISCUSSION TITLE
Is there some kind of record of what errors Pocketbase will return in different cases when extending with Go?

### DISCUSSION THREAD

**Discussion by emmalexandria** [pocketbase/pocketbase#4013] [Q&A]
There are certain errors I'd like to conditionally ignore as expected errors (e.g errors with constraints which I expect to not be met). I'm sure I could go through a trial and error process to determine this, but I was wondering if I missed some part of the documentation.

**Comment by ganigeorgiev** [comment]
No, there is no such list at the moment.

There are plans to document the errors for the web APIs to make it easier for translations, but I don't think we can do the same with every returned internal Go error because it could vary too much.
Additionally, because we support both the CGO and pure Go SQLite drivers, note that there could be some minor differences to the returned db errors and their messages format.
